js/classes/entities/foreground/hud.js

```diff
@@ -541,4 +538,35 @@ class HudHealthBar {
             (HudHealthBar.SIZE.x - HudHealthBar.PADDING * 2) * CHAD.health / Chad.MAX_HEALTH,
             HudHealthBar.BAR_HEIGHT - HudHealthBar.PADDING * 2);
     };
+}
+
+/**
+ * Chad's head icon component.
+ * 
+ * @author Trae Claar
+ */
+class ChadHead {
+    /** 
+     * Constructor for the ChadHead.
+     */
+    constructor() {
+        this.animator = new Animator(Chad.SPRITESHEET,
```

Comment:
Nice, ChadHead is its own class? What if we made it animate  the head with the hair like Chad to show his movement state? Super unnecessary, just a fun idea...

Reply:
That would be cool!